Data/Sys/GameSettings/SD2J01.ini

```diff
@@ -26,8 +26,8 @@ PH_ZNear =
 PH_ZFar =
 
 [Video_Settings]
-SafeTextureCacheColorSamples = 512
```

Comment:
Intro videos are not smooth and are erratic without it, it shouldn't be removed imo.